### ISSUE TITLE
Boogie error: Unsupported modifier invocation

### ISSUE THREAD

**Issue by dddejan** [SRI-CSL/solidity#22]
```solidity
contract A {
  constructor() m1 public { }
  modifier m1 { _; }
}
contract B is A {
  modifier m2 { _; }
  constructor() A() m1 m2 public {  }
}
```

Error:
```
./solc-verify.py test/libsolidity/syntaxTests/inheritance/modifiers_in_constructor_context.sol --output .
Error while running compiler, details:
Warning: This is a pre-release compiler version, please do not use it in production.
test/libsolidity/syntaxTests/inheritance/modifiers_in_constructor_context.sol:2:1: Warning: Source file does not specify required compiler version!
contract A {
^ (Relevant source part starts here and spans across multiple lines).

======= Converting to Boogie IVL =======

======= test/libsolidity/syntaxTests/inheritance/modifiers_in_constructor_context.sol =======
test/libsolidity/syntaxTests/inheritance/modifiers_in_constructor_context.sol:8:17: Boogie error: Unsupported modifier invocation
  constructor() A() m1 m2 public {  }
                ^-^
```

**Comment by hajduakos** [comment]
Added a more meaningful error message in 6e71acf

**Comment by hajduakos** [comment]
Fixed in de5efb6. Didn't add this example as a test case because `BaseConstructorWithModifiers.sol` covers it.
